e2ev2/tests/workspace-create.test.js

```diff
@@ -58,7 +58,9 @@ browserTest('create a workspace', async browser => {
   const confirmButton =
     await page.waitForSelector('[role="button"][aria-label="Confirm Delete"]')
   await expect(confirmButton.evaluate(n => n.style.cursor)).resolves.toBe('not-allowed')
-  await page.type('[role="dialog"] input[placeholder="type DELETE to confirm"]', 'delete')
```

Comment:
We updated the UI so this dialog does not appear until we have checked for the presence of apps (and confirmed there are none)

So: we need to wait for it.